tests/testthat/test_stats_wrapper.R

```diff
@@ -91,6 +91,18 @@ test_that("test do_cor without val", {
   expect_equal(result[[3]][1:2], c(1, -1))
 })
 
+test_that("do_svd.kv with NA value", {
+  data <- data.frame(row = c(1, 1, 2, 2, 3,3),
+                     col = rep(c(1,2), 3),
+                     val = seq(6)) %>%
+    dplyr::slice(-3)
```

Comment:
I didn't know this trick of slice with negative number. I would appreciate a comment.